infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -81,11 +84,17 @@
      */
     public SequencerServerCache(int cacheSize, long maxConflictNewSequencer) {
         this.cacheSize = cacheSize;
-        conflictKeys = new HashMap();
+
         cacheEntries = new PriorityQueue(cacheSize, Comparator.comparingLong
-                (a -> ((ConflictTxStream)a).txVersion));
+                (a -> ((ConflictTxStream) a).txVersion));
         maxConflictWildcard = maxConflictNewSequencer;
         this.maxConflictNewSequencer = maxConflictNewSequencer;
+        conflictKeys = MeterRegistryProvider
+                .getInstance()
+                .map(registry ->
+                        registry.gauge(conflictKeysCounterName, Collections.emptyList(),
+                                new HashMap<ConflictTxStream, Long>(), HashMap::size))
+                .orElse(new HashMap<>());
```

Comment:
We need to capture stats on the window size and number of evictions per trim call.

Reply:
Added